ActiveLabeler-main/ActiveLabelerModels/ClassifierModel.py

```diff
@@ -0,0 +1,46 @@
+import torch
```

Comment:
Consider maintaining a consistent folder and file naming structure. Typically in Python, file and folder names are all lowercase and underscores are used to separate words.